Lombiq.NodeJs.Extensions/scripts/validate-config.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @summary Validates the Node.js Extensions configuration object.
+ * @description This script validates a configuration object against the format provided in @example.
+ * @example "nodejsExtensions": {
+ *            "assetsToCopy": [
+ *              {
+ *                "sources": [ "Assets/Images" ],
+ *                "target": "wwwroot/images"
+ *              },
+ *              {
+ *                "sources": [
+ *                  "node_modules/marvelous/dist"
+ *                  "node_modules/wonderful/bin"
+ *                ],
+ *                "pattern": "*",
+ *                "target": "wwwroot"
+ *              }
+ *              // more groups, if needed
+ *            ],
+ *            "scripts": {
+ *              "source": "CustomJsFolder",
+ *              "target": "wwwCustomRoot/jay-es"
+ *            },
+ *            "styles": {
+ *              "source": "NonDefaultScssFolder",
+ *              "target": "wwwwNonDefaultCssFolder"
+ *            }
+ *          }
+ */
+function logError(message) { process.stderr.write(`ERROR: ${message}\n`); }
```

Comment:
Once OSOE-92 is merged, please replace this with `const { handleErrorObject } = require('./handle-error');`. Same with the other `process.stderr.write` in get-path.js and get-config.js.